tools/gentest.ml

```diff
@@ -152,6 +177,11 @@ end = struct
           pp_expected_output tst
           pp_output tst
     in
+    let accepted_ae_exit_code =
+      if tst.params.ae_should_succeed then
+        "0"
+      else "(or 1 123 124 125 142)"
```

Comment:
Probably only `1` should be allowed here because it is the only one that actually indicates an error of the solver — `142` is a timeout (very different!), and `123`, `124` and `125` are for command-line parsing errors which shouldn't occur at all in tests.